Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -675,15 +683,10 @@ public virtual void OnApplicationQuit()
         /// </summary>
         public virtual void ConfigureServerFrameRate()
         {
-            // set a fixed tick rate instead of updating as often as possible
-            // * if not in Editor (it doesn't work in the Editor)
-            // * if not in Host mode
-#if !UNITY_EDITOR
-            if (!NetworkClient.active && isHeadless)
-            {
-                Application.targetFrameRate = serverTickRate;
-                if (logger.logEnabled) logger.Log("Server Tick Rate set to: " + Application.targetFrameRate + " Hz.");
-            }
+            // only set framerate for server build
+#if UNITY_SERVER
+            Application.targetFrameRate = serverTickRate;
```

Comment:
please check if we don't get warnings when doing this in editor, since we had #unity_editor before

Reply:
`UNITY_SERVER` shouldn't be set in the editor so this should be fine

Reply:
It won’t be. UNITY_SERVER is a standalone only compile define.